stable_baselines3/common/base_class.py

```diff
@@ -130,7 +130,7 @@ def __init__(
         self.tensorboard_log = tensorboard_log
         self.lr_schedule = None  # type: Optional[Schedule]
         self._last_obs = None  # type: Optional[np.ndarray]
-        self._last_dones = None  # type: Optional[np.ndarray]
+        self._episode_starts = None  # type: Optional[np.ndarray]
```

Comment:
I would name this `_last_episode_starts` for consistency, bit verbose, but at least tells its meaning :)